x/mongo/driver/operation.go

```diff
@@ -309,6 +310,13 @@ func (op Operation) Validate() error {
 	return nil
 }
 
+var writeBuffers = sync.Pool{
+	New: func() interface{} {
+		b := make([]byte, 1024) // Start with 1kb buffers.
+		return &b
+	},
+}
+
 // Execute runs this operation. The scratch parameter will be used and overwritten (potentially many
```

Comment:
Does the `scratch` argument serve any purpose anymore? If not, I suggest removing it, and updating this comment.